packages/svelte-ux/src/lib/components/Toast.svelte

```diff
@@ -0,0 +1,14 @@
+<script lang="ts">
+  import toastStore from '$lib/stores/toastStore.js';
+  import Notification from './Notification.svelte';
+</script>
+
+<div class="fixed bottom-0 left-1/2 translate-x-[-50%] z-50 gap-y-4">
```

Comment:
I could see us adding a `placement` prop to control which of the 8 corners/edges to place the notifications (at least `top-left`, `top`, `top-right`, `bottom-left`, `bottom`, `bottom-right`).

you can see some examples of this in [Drawer](https://github.com/techniq/svelte-ux/blob/main/packages/svelte-ux/src/lib/components/Drawer.svelte#L76-L83) and [Badge](https://github.com/techniq/svelte-ux/blob/main/packages/svelte-ux/src/lib/components/Badge.svelte#L12)

Reply:
It would also be good if `Toast` supported class overrides, either directly (`<Toast classes={...}>`) or via [settings](https://svelte-ux.techniq.dev/customization#settings).

To support this, you would need to add a `classes` prop and also getting the classes defined in `settings(...)` using...

```svelte
<script>
  ...

  export let classes: {
    root?: string;
    actions?: string;
    // ...any other internal element/component you might want to pass a class to
  } = {};
  const settingsClasses = getComponentClasses('Toast');
</script>
```

and change...

```svelte
<div class="fixed bottom-0 left-1/2 translate-x-[-50%] z-50 gap-y-4">
```

to

```svelte
<div
    class={cls(
      'Toast',
      'fixed bottom-0 left-1/2 translate-x-[-50%] z-50 gap-y-4',
      settingsClasses.root,
      classes.root,
      $$props.class
    )}
>